### PR TITLE
Standardization of kubernetes.md

### LINKED ISSUE #9199: [Docs] Standardization of Supported platform
### Current State
Kubernetes docs is not standardized similar to other supproted platform docs

### Desired State
Standardisation of Kubernetes docs

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by aboobakersiddiqr63**
**Notes for Reviewers**

This PR fixes #9199 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ✓] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f4933e3475baf6f05bac29bdd6a395eb73a730c2 at: https://6539227ff9afd6009479e8d5--meshery-docs-preview.netlify.app

File: docs/pages/installation/kinD.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Meshery should now be running in your kinD cluster and Meshery UI should be acce
 
 Once kinD cluster is configured as current cluster-context, execute the below command.
 
+Before executing the below command, go to ~/.meshery/config.yaml and ensure that current platform is set to kubernetes.
```
Comment (alphaX86) [inline docs/pages/installation/kinD.md:72]:
```suggestion
Before executing the below command, go to `~/.meshery/config.yaml` and ensure that current platform is set to kubernetes.
```

**Review by alphaX86 — CHANGES_REQUESTED** [review]

File: docs/pages/installation/kubernetes.md [hunk 2]
```diff
@@ -10,164 +10,72 @@ list: include
...
-</code>
-</pre>
-
-</details>
-
-Deploy Meshery to your Kubernetes cluster by executing:
-
-{% capture code_content %}$ mesheryctl system start{% endcapture %}
-{% include code.html code=code_content %}
+Manage your kubernetes clusters with Meshery. Deploy Meshery in kubernetes [in-cluster](#in-cluster-installation) or outside of kubernetes [out-of-cluster](#out-of-cluster-installation). **_Note: It is advisable to [Install Meshery in your kubernetes clusters](#install-meshery-into-your-kubernetes-cluster)_**
```
[the remark this answers is not in the document]
Comment (aboobakersiddiqr63) [reply docs/pages/installation/kubernetes.md:15]: @alphaX86 thanks for pointing it out will fix it

[the remark this answers is not in the document]
Comment (aboobakersiddiqr63) [reply docs/pages/installation/kubernetes.md:15]: @alphaX86 fixed the issue :)

**Comment by l5io** [comment]
🚀 Preview for commit 9f54da9c2d7d6ad47e731f5e9799d4ed84779ba2 at: https://653958131fad360f95d184de--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 54ded6515921a1c9f159c5ccaf7a99df92b1a102 at: https://653a96730a2f791661d5597c--meshery-docs-preview.netlify.app

**Review by alphaX86 — APPROVED** [review]
LGTM 👍

**Pull request merged**